docker/frontend/README.md

```diff
@@ -4,18 +4,13 @@
 Refer to [DataHub Frontend Service](../../datahub-frontend) to have a quick understanding of the architecture and 
 responsibility of this service for the DataHub.
 
-## Build
+## Build & Run
 ```
-docker image build -t linkedin/datahub-frontend -f docker/frontend/Dockerfile .
+cd docker/frontend && docker-compose up --build
 ```
-This command will build and deploy the image in your local store.
+This command will rebuild the docker image and start a container based on the image.
 
-## Run container
-```
-cd docker/frontend && docker-compose pull && docker-compose up
-```
-This command will start the container. If you have the image available in your local store, this image will be used
-for the container otherwise it will download the `latest` image from Docker Hub and then start that.
+To start a container using an existing image, run the same command without the `--build` flag.
```

Comment:
I believe the intention of these Dockerfiles is for local development, instead of quickstart. As such pulling the latest image from docker hub is probably not a common operation in this case?